src/js/newsletter/components/ConditionsCheckbox.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+
+export default function ConditionsCheckbox(props) {
+  return (
+    <div className="control">
+      <input
+        {...props}
+        type="checkbox"
+        id="conditions"
```

Comment:
that will not work properly, as an id should be unique

Reply:
Ah I'm sorry. That was stupid.
Before introducing a new dependency to generate unique IDs, I rethought why it is there. The only advantage of such link I remember would be the possibility to click the label to check the box. But since our label includes clickable web-links, this is not useful at all. So to keep it as simple as possible for the use-case I would just remove the association.